.storybook/webpack.config.js

```diff
@@ -4,9 +4,6 @@
 
 const path = require('path');
 
-const {
```

Comment:
Turns out we missed deleting this line when removing some of our icon loading logic a while back.